include/vcpkg/platform-expression.h

```diff
@@ -25,14 +25,14 @@ namespace vcpkg::PlatformExpression
         static Expr Or(std::vector<Expr>&& exprs);
 
         // since ExprImpl is not yet defined, we need to define the ctor and dtor in the C++ file
-        Expr(); // always true
+        Expr() noexcept; // always true
         Expr(const Expr&);
-        Expr(Expr&&);
+        Expr(Expr&&) noexcept;
         Expr& operator=(const Expr& e);
-        Expr& operator=(Expr&&);
+        Expr& operator=(Expr&&) noexcept;
 
         explicit Expr(std::unique_ptr<detail::ExprImpl>&& e);
-        ~Expr();
+        ~Expr() noexcept;
```

Comment:
```suggestion
        ~Expr();
```

Destructors default to noexcept.